docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+services:
+  app-backend:
+    restart: always
+    image: t-learn/backend:latest
```

Comment:
not sure the image name matches the one created by Quarkus (Quarkus also uses versions)